Description: Append 'gmt' to the man page sections to differentiate the GMT specific manuals.
Author: Bas Couwenberg <sebastic@debian.org>
Bug-Debian: https://bugs.debian.org/790524
Forwarded: not-needed

--- a/doc/rst/conf.py.in
+++ b/doc/rst/conf.py.in
@@ -231,133 +231,133 @@ latex_elements = {
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-('blockmean', 'blockmean', 'Block average (x, y, z) data tables by L2 norm', '', 1),
-('blockmedian', 'blockmedian', 'Block average (x, y, z) data tables by L1 norm', '', 1),
-('blockmode', 'blockmode', 'Block average (x, y, z) data tables by mode estimation', '', 1),
-('filter1d', 'filter1d', 'Do time domain filtering of 1-D data tables', '', 1),
-('fitcircle', 'fitcircle', 'find mean position and pole of best-fit great [or small] circle to points on a sphere.', '', 1),
-('gmt_shell_functions.sh', 'gmt_shell_functions.sh', 'Practical functions to be used in GMT bourne shell scripts', '', 1),
-('gmt.conf', 'gmt.conf', 'Configuration for GMT', '', 5),
-('gmt', 'gmt', 'The Generic Mapping Tools data processing and display software package', '', 1),
-('gmt2kml', 'gmt2kml', 'Convert GMT data tables to KML files for Google Earth', '', 1),
-('gmt5syntax', 'gmt5syntax', 'Convert old GMT script to use new \'gmt <module>\' syntax', '', 1),
-('gmtcolors', 'gmtcolors', 'Explanation of color codes in GMT', '', 5),
-('gmtconnect', 'gmtconnect', 'Connect individual lines whose end points match within tolerance', '', 1),
-('gmtconvert', 'gmtconvert', 'Convert, Paste, and/or Extract columns from data tables', '', 1),
-('gmtdefaults', 'gmtdefaults', 'List current GMT default parameters', '', 1),
-('gmtget', 'gmtget', 'Get individual GMT default parameters', '', 1),
-('gmtlogo', 'gmtlogo', 'Adding a GMT graphics logo overlay to an illustration', '', 1),
-('gmtinfo', 'gmtinfo', 'Return information about data tables', '', 1),
-('gmtmath', 'gmtmath', 'Reverse Polish Notation (RPN) calculator for data tables', '', 1),
-('gmtselect', 'gmtselect', 'Select data table subsets based on multiple spatial criteria', '', 1),
-('gmtset', 'gmtset', 'Change individual GMT default parameters', '', 1),
-('gmtsimplify', 'gmtsimplify', 'Line reduction using the Douglas-Peucker algorithm', '', 1),
-('gmtspatial', 'gmtspatial', 'Do geospatial operations on lines and polygons', '', 1),
-('gmtswitch', 'gmtswitch', 'Switching between different GMT versions', '', 1),
-('gmtvector', 'gmtvector', 'Basic manipulation of Cartesian vectors', '', 1),
-('gmtwhich', 'gmtwhich', 'Find full path to specified files', '', 1),
-('grd2cpt', 'grd2cpt', 'Make linear or histogram-equalized color palette table from grid', '', 1),
-('grd2rgb', 'grd2rgb', 'Write r/g/b grid files from a grid file, a raw RGB file, or SUN rasterfile', '', 1),
-('grd2xyz', 'grd2xyz', 'Convert grid file to data table', '', 1),
-('grdblend', 'grdblend', 'Blend several partially over-lapping grids into one large grid', '', 1),
-('grdclip', 'grdclip', 'Clip the range of grids', '', 1),
-('grdcontour', 'grdcontour', 'Make contour map using a grid', '', 1),
-('grdcut', 'grdcut', 'Extract subregion from a grid', '', 1),
-('grdedit', 'grdedit', 'Modify header or content of a grid', '', 1),
-('grdfft', 'grdfft', 'Do mathematical operations on grids in the wavenumber (or frequency) domain', '', 1),
-('grdfilter', 'grdfilter', 'Filter a grid in the space (or time) domain', '', 1),
-('grdgradient', 'grdgradient', 'Compute directional derivative or gradient from a grid', '', 1),
-('grdhisteq', 'grdhisteq', 'Perform histogram equalization for a grid', '', 1),
-('grdimage', 'grdimage', 'Project grids or images and plot them on maps', '', 1),
-('grdinfo', 'grdinfo', 'Extract information from grids', '', 1),
-('grdlandmask', 'grdlandmask', '"Create a ""wet-dry"" mask grid from shoreline data base"', '', 1),
-('grdmask', 'grdmask', 'Create mask grid from polygons or point coverage', '', 1),
-('grdmath', 'grdmath', 'Reverse Polish Notation (RPN) calculator for grids (element by element)', '', 1),
-('grdpaste', 'grdpaste', 'Join two grids along their common edge', '', 1),
-('grdproject', 'grdproject', 'Forward and inverse map transformation of grids', '', 1),
-('grdraster', 'grdraster', 'Extract subregion from a binary raster and save as a GMT grid', '', 1),
-('grdreformat', 'grdreformat', 'Convert between different grid formats', '', 1),
-('grdsample', 'grdsample', 'Resample a grid onto a new lattice', '', 1),
-('grdtrack', 'grdtrack', 'Sample grids at specified (x,y) locations', '', 1),
-('grdtrend', 'grdtrend', 'Fit trend surface to grids and compute residuals', '', 1),
-('grdvector', 'grdvector', 'Plot vector field from two component grids', '', 1),
-('grdview', 'grdview', 'Create 3-D perspective image or surface mesh from a grid', '', 1),
-('grdvolume', 'grdvolume', 'Calculate grid volume and area constrained by a contour', '', 1),
-('greenspline', 'greenspline', 'Interpolate using Green\'s functions for splines in 1-3 dimensions', '', 1),
-('isogmt', 'isogmt', 'Run GMT command or script in isolation mode', '', 1),
-('kml2gmt', 'kml2gmt', 'Extract GMT table data from Google Earth KML files', '', 1),
-('makecpt', 'makecpt', 'Make GMT color palette tables', '', 1),
-('mapproject', 'mapproject', 'Do forward and inverse map transformations, datum conversions and geodesy', '', 1),
-('nearneighbor', 'nearneighbor', '"Grid table data using a ""Nearest neighbor"" algorithm"', '', 1),
-('project', 'project', 'Project table data onto lines or great circles, generate tracks, or translate coordinates', '', 1),
-('ps2raster', 'ps2raster', 'Convert [E]PS file(s) to other formats using GhostScript', '', 1),
-('psbasemap', 'psbasemap', 'Plot PostScript base maps', '', 1),
-('psclip', 'psclip', 'Initialize or terminate polygonal clip paths', '', 1),
-('pscoast', 'pscoast', 'Plot continents, shorelines, rivers, and borders on maps', '', 1),
-('pscontour', 'pscontour', 'Contour table data by direct triangulation [method]', '', 1),
-('pshistogram', 'pshistogram', 'Calculate and plot histograms', '', 1),
-('psimage', 'psimage', 'Place images or EPS files on maps', '', 1),
-('pslegend', 'pslegend', 'Plot legends on maps', '', 1),
-('pslib', 'pslib', 'A PostScript based plotting library', '', 3),
-('psmask', 'psmask', 'Use data tables to clip or mask map areas with no coverage', '', 1),
-('psrose', 'psrose', 'Plot a polar histogram (rose, sector, windrose diagrams)', '', 1),
-('psscale', 'psscale', 'Plot a gray or color scale-bar on maps', '', 1),
-('pstext', 'pstext', 'Plot or typeset text on maps', '', 1),
-('pswiggle', 'pswiggle', 'Plot z = f(x,y) anomalies along tracks', '', 1),
-('psxy', 'psxy', 'Plot lines, polygons, and symbols on maps', '', 1),
-('psxyz', 'psxyz', 'Plot lines, polygons, and symbols in 3-D', '', 1),
-('sample1d', 'sample1d', 'Resample 1-D table data using splines', '', 1),
-('spectrum1d', 'spectrum1d', 'Compute auto- [and cross- ] spectra from one [or two] time-series', '', 1),
-('sph2grd', 'sph2grd', 'Compute grid from spherical harmonic coefficients', '', 1),
-('sphdistance', 'sphdistance', 'Make grid of distances to nearest points on a sphere', '', 1),
-('sphinterpolate', 'sphinterpolate', 'Spherical gridding in tension of data on a sphere', '', 1),
-('sphtriangulate', 'sphtriangulate', 'Delaunay or Voronoi construction of spherical lon,lat data', '', 1),
-('splitxyz', 'splitxyz', 'Split xyz[dh] data tables into individual segments', '', 1),
-('surface', 'surface', 'Grid table data using adjustable tension continuous curvature splines', '', 1),
-('trend1d', 'trend1d', 'Fit a [weighted] [robust] polynomial [or Fourier] model for y = f(x) to xy[w] data', '', 1),
-('trend2d', 'trend2d', 'Fit a [weighted] [robust] polynomial model for z = f(x,y) to xyz[w] data', '', 1),
-('triangulate', 'triangulate', 'Do optimal (Delaunay) triangulation and gridding of Cartesian table data [method]', '', 1),
-('xyz2grd', 'xyz2grd', 'Convert data table to a grid file', '', 1),
-('supplements/gshhg/gshhg', 'gshhg', 'Extract data tables from binary GSHHG or WDBII data files', '', 1),
-('supplements/img/img2grd', 'img2grd', 'Extract subset of img file in Mercator or Geographic format', '', 1),
-('supplements/meca/pscoupe', 'pscoupe', 'Plot cross-sections of focal mechanisms', '', 1),
-('supplements/meca/psmeca', 'psmeca', 'Plot focal mechanisms on maps', '', 1),
-('supplements/meca/pspolar', 'pspolar', 'Plot polarities on the inferior focal half-sphere on maps', '', 1),
-('supplements/meca/psvelo', 'psvelo', 'Plot velocity vectors, crosses, and wedges on maps', '', 1),
-('supplements/mgd77/mgd77convert', 'mgd77convert', 'Convert MGD77 data to other file formats', '', 1),
-('supplements/mgd77/mgd77info', 'mgd77info', 'Extract information about MGD77 files', '', 1),
-('supplements/mgd77/mgd77list', 'mgd77list', 'Extract data from MGD77 files', '', 1),
-('supplements/mgd77/mgd77magref', 'mgd77magref', 'Evaluate the IGRF or CM4 magnetic field models', '', 1),
-('supplements/mgd77/mgd77manage', 'mgd77manage', 'Manage the content of MGD77+ files', '', 1),
-('supplements/mgd77/mgd77path', 'mgd77path', 'Return paths to MGD77 cruises and directories', '', 1),
-('supplements/mgd77/mgd77sniffer', 'mgd77sniffer', 'Along-track quality control of MGD77 cruises', '', 1),
-('supplements/mgd77/mgd77track', 'mgd77track', 'Plot track-line map of MGD77 cruises', '', 1),
-('supplements/misc/dimfilter', 'dimfilter', 'Directional filtering of 2-D gridded files in the space (or time) domain', '', 1),
-('supplements/potential/gmtgravmag3d', 'gmtgravmag3d', 'Compute the gravity/magnetic effect of a body by the method of Okabe', '', 1),
-('supplements/potential/gravfft', 'gravfft', 'Compute gravitational attraction of 3-D surfaces in the wavenumber (or frequency) domain', '', 1),
-('supplements/potential/grdgravmag3d', 'grdgravmag3d', 'Compute the gravity effect of a grid by the method of Okabe', '', 1),
-('supplements/potential/grdredpol', 'grdredpol', 'Compute the Continuous Reduction To the Pole, AKA differential RTP.', '', 1),
-('supplements/potential/grdseamount', 'grdseamount', 'Compute synthetic seamount (Gaussian or cone, circular or elliptical) bathymetry', '', 1),
-('supplements/segy/pssegy', 'pssegy', 'Plot a SEGY file on a map', '', 1),
-('supplements/segy/pssegyz', 'pssegyz', 'Create imagemasked postscript from SEGY file', '', 1),
-('supplements/segy/segy2grd', 'segy2grd', 'Converting SEGY data to a GMT grid', '', 1),
-('supplements/spotter/backtracker', 'backtracker', 'Generate forward and backward flowlines and hotspot tracks', '', 1),
-('supplements/spotter/grdpmodeler', 'grdpmodeler', 'Evaluate a plate model on a geographic grid', '', 1),
-('supplements/spotter/grdrotater', 'grdrotater', 'Finite rotation reconstruction of geographic grid', '', 1),
-('supplements/spotter/grdspotter', 'grdspotter', 'Create CVA image from a gravity or topography grid', '', 1),
-('supplements/spotter/hotspotter', 'hotspotter', 'Create CVA image from seamount locations', '', 1),
-('supplements/spotter/originator', 'originator', 'Associate seamounts with nearest hotspot point sources', '', 1),
-('supplements/spotter/rotconverter', 'rotconverter', 'Manipulate total reconstruction and stage rotations', '', 1),
-('supplements/x2sys/x2sys_binlist', 'x2sys_binlist', 'Create bin index listing from track data files', '', 1),
-('supplements/x2sys/x2sys_cross', 'x2sys_cross', 'Calculate crossovers between track data files', '', 1),
-('supplements/x2sys/x2sys_datalist', 'x2sys_datalist', 'Extract content of track data files', '', 1),
-('supplements/x2sys/x2sys_get', 'x2sys_get', 'Get track listing from the x2sys track index databases', '', 1),
-('supplements/x2sys/x2sys_init', 'x2sys_init', 'Initialize a new x2sys track database', '', 1),
-('supplements/x2sys/x2sys_list', 'x2sys_list', 'Extract subset from crossover data base', '', 1),
-('supplements/x2sys/x2sys_merge', 'x2sys_merge', 'Merge an updated COEs table (smaller) into the main table (bigger)', '', 1),
-('supplements/x2sys/x2sys_put', 'x2sys_put', 'Update track index database from track bin file', '', 1),
-('supplements/x2sys/x2sys_report', 'x2sys_report', 'Report statistics from crossover data base', '', 1),
-('supplements/x2sys/x2sys_solve', 'x2sys_solve', 'Determine least-squares systematic correction from crossovers', '', 1)
+('blockmean', 'blockmean', 'Block average (x, y, z) data tables by L2 norm', '', '1gmt'),
+('blockmedian', 'blockmedian', 'Block average (x, y, z) data tables by L1 norm', '', '1gmt'),
+('blockmode', 'blockmode', 'Block average (x, y, z) data tables by mode estimation', '', '1gmt'),
+('filter1d', 'filter1d', 'Do time domain filtering of 1-D data tables', '', '1gmt'),
+('fitcircle', 'fitcircle', 'find mean position and pole of best-fit great [or small] circle to points on a sphere.', '', '1gmt'),
+('gmt_shell_functions.sh', 'gmt_shell_functions.sh', 'Practical functions to be used in GMT bourne shell scripts', '', '1gmt'),
+('gmt.conf', 'gmt.conf', 'Configuration for GMT', '', '5gmt'),
+('gmt', 'gmt', 'The Generic Mapping Tools data processing and display software package', '', '1gmt'),
+('gmt2kml', 'gmt2kml', 'Convert GMT data tables to KML files for Google Earth', '', '1gmt'),
+('gmt5syntax', 'gmt5syntax', 'Convert old GMT script to use new \'gmt <module>\' syntax', '', '1gmt'),
+('gmtcolors', 'gmtcolors', 'Explanation of color codes in GMT', '', '5gmt'),
+('gmtconnect', 'gmtconnect', 'Connect individual lines whose end points match within tolerance', '', '1gmt'),
+('gmtconvert', 'gmtconvert', 'Convert, Paste, and/or Extract columns from data tables', '', '1gmt'),
+('gmtdefaults', 'gmtdefaults', 'List current GMT default parameters', '', '1gmt'),
+('gmtget', 'gmtget', 'Get individual GMT default parameters', '', '1gmt'),
+('gmtlogo', 'gmtlogo', 'Adding a GMT graphics logo overlay to an illustration', '', '1gmt'),
+('gmtinfo', 'gmtinfo', 'Return information about data tables', '', '1gmt'),
+('gmtmath', 'gmtmath', 'Reverse Polish Notation (RPN) calculator for data tables', '', '1gmt'),
+('gmtselect', 'gmtselect', 'Select data table subsets based on multiple spatial criteria', '', '1gmt'),
+('gmtset', 'gmtset', 'Change individual GMT default parameters', '', '1gmt'),
+('gmtsimplify', 'gmtsimplify', 'Line reduction using the Douglas-Peucker algorithm', '', '1gmt'),
+('gmtspatial', 'gmtspatial', 'Do geospatial operations on lines and polygons', '', '1gmt'),
+('gmtswitch', 'gmtswitch', 'Switching between different GMT versions', '', '1gmt'),
+('gmtvector', 'gmtvector', 'Basic manipulation of Cartesian vectors', '', '1gmt'),
+('gmtwhich', 'gmtwhich', 'Find full path to specified files', '', '1gmt'),
+('grd2cpt', 'grd2cpt', 'Make linear or histogram-equalized color palette table from grid', '', '1gmt'),
+('grd2rgb', 'grd2rgb', 'Write r/g/b grid files from a grid file, a raw RGB file, or SUN rasterfile', '', '1gmt'),
+('grd2xyz', 'grd2xyz', 'Convert grid file to data table', '', '1gmt'),
+('grdblend', 'grdblend', 'Blend several partially over-lapping grids into one large grid', '', '1gmt'),
+('grdclip', 'grdclip', 'Clip the range of grids', '', '1gmt'),
+('grdcontour', 'grdcontour', 'Make contour map using a grid', '', '1gmt'),
+('grdcut', 'grdcut', 'Extract subregion from a grid', '', '1gmt'),
+('grdedit', 'grdedit', 'Modify header or content of a grid', '', '1gmt'),
+('grdfft', 'grdfft', 'Do mathematical operations on grids in the wavenumber (or frequency) domain', '', '1gmt'),
+('grdfilter', 'grdfilter', 'Filter a grid in the space (or time) domain', '', '1gmt'),
+('grdgradient', 'grdgradient', 'Compute directional derivative or gradient from a grid', '', '1gmt'),
+('grdhisteq', 'grdhisteq', 'Perform histogram equalization for a grid', '', '1gmt'),
+('grdimage', 'grdimage', 'Project grids or images and plot them on maps', '', '1gmt'),
+('grdinfo', 'grdinfo', 'Extract information from grids', '', '1gmt'),
+('grdlandmask', 'grdlandmask', '"Create a ""wet-dry"" mask grid from shoreline data base"', '', '1gmt'),
+('grdmask', 'grdmask', 'Create mask grid from polygons or point coverage', '', '1gmt'),
+('grdmath', 'grdmath', 'Reverse Polish Notation (RPN) calculator for grids (element by element)', '', '1gmt'),
+('grdpaste', 'grdpaste', 'Join two grids along their common edge', '', '1gmt'),
+('grdproject', 'grdproject', 'Forward and inverse map transformation of grids', '', '1gmt'),
+('grdraster', 'grdraster', 'Extract subregion from a binary raster and save as a GMT grid', '', '1gmt'),
+('grdreformat', 'grdreformat', 'Convert between different grid formats', '', '1gmt'),
+('grdsample', 'grdsample', 'Resample a grid onto a new lattice', '', '1gmt'),
+('grdtrack', 'grdtrack', 'Sample grids at specified (x,y) locations', '', '1gmt'),
+('grdtrend', 'grdtrend', 'Fit trend surface to grids and compute residuals', '', '1gmt'),
+('grdvector', 'grdvector', 'Plot vector field from two component grids', '', '1gmt'),
+('grdview', 'grdview', 'Create 3-D perspective image or surface mesh from a grid', '', '1gmt'),
+('grdvolume', 'grdvolume', 'Calculate grid volume and area constrained by a contour', '', '1gmt'),
+('greenspline', 'greenspline', 'Interpolate using Green\'s functions for splines in 1-3 dimensions', '', '1gmt'),
+('isogmt', 'isogmt', 'Run GMT command or script in isolation mode', '', '1gmt'),
+('kml2gmt', 'kml2gmt', 'Extract GMT table data from Google Earth KML files', '', '1gmt'),
+('makecpt', 'makecpt', 'Make GMT color palette tables', '', '1gmt'),
+('mapproject', 'mapproject', 'Do forward and inverse map transformations, datum conversions and geodesy', '', '1gmt'),
+('nearneighbor', 'nearneighbor', '"Grid table data using a ""Nearest neighbor"" algorithm"', '', '1gmt'),
+('project', 'project', 'Project table data onto lines or great circles, generate tracks, or translate coordinates', '', '1gmt'),
+('ps2raster', 'ps2raster', 'Convert [E]PS file(s) to other formats using GhostScript', '', '1gmt'),
+('psbasemap', 'psbasemap', 'Plot PostScript base maps', '', '1gmt'),
+('psclip', 'psclip', 'Initialize or terminate polygonal clip paths', '', '1gmt'),
+('pscoast', 'pscoast', 'Plot continents, shorelines, rivers, and borders on maps', '', '1gmt'),
+('pscontour', 'pscontour', 'Contour table data by direct triangulation [method]', '', '1gmt'),
+('pshistogram', 'pshistogram', 'Calculate and plot histograms', '', '1gmt'),
+('psimage', 'psimage', 'Place images or EPS files on maps', '', '1gmt'),
+('pslegend', 'pslegend', 'Plot legends on maps', '', '1gmt'),
+('pslib', 'pslib', 'A PostScript based plotting library', '', '3gmt'),
+('psmask', 'psmask', 'Use data tables to clip or mask map areas with no coverage', '', '1gmt'),
+('psrose', 'psrose', 'Plot a polar histogram (rose, sector, windrose diagrams)', '', '1gmt'),
+('psscale', 'psscale', 'Plot a gray or color scale-bar on maps', '', '1gmt'),
+('pstext', 'pstext', 'Plot or typeset text on maps', '', '1gmt'),
+('pswiggle', 'pswiggle', 'Plot z = f(x,y) anomalies along tracks', '', '1gmt'),
+('psxy', 'psxy', 'Plot lines, polygons, and symbols on maps', '', '1gmt'),
+('psxyz', 'psxyz', 'Plot lines, polygons, and symbols in 3-D', '', '1gmt'),
+('sample1d', 'sample1d', 'Resample 1-D table data using splines', '', '1gmt'),
+('spectrum1d', 'spectrum1d', 'Compute auto- [and cross- ] spectra from one [or two] time-series', '', '1gmt'),
+('sph2grd', 'sph2grd', 'Compute grid from spherical harmonic coefficients', '', '1gmt'),
+('sphdistance', 'sphdistance', 'Make grid of distances to nearest points on a sphere', '', '1gmt'),
+('sphinterpolate', 'sphinterpolate', 'Spherical gridding in tension of data on a sphere', '', '1gmt'),
+('sphtriangulate', 'sphtriangulate', 'Delaunay or Voronoi construction of spherical lon,lat data', '', '1gmt'),
+('splitxyz', 'splitxyz', 'Split xyz[dh] data tables into individual segments', '', '1gmt'),
+('surface', 'surface', 'Grid table data using adjustable tension continuous curvature splines', '', '1gmt'),
+('trend1d', 'trend1d', 'Fit a [weighted] [robust] polynomial [or Fourier] model for y = f(x) to xy[w] data', '', '1gmt'),
+('trend2d', 'trend2d', 'Fit a [weighted] [robust] polynomial model for z = f(x,y) to xyz[w] data', '', '1gmt'),
+('triangulate', 'triangulate', 'Do optimal (Delaunay) triangulation and gridding of Cartesian table data [method]', '', '1gmt'),
+('xyz2grd', 'xyz2grd', 'Convert data table to a grid file', '', '1gmt'),
+('supplements/gshhg/gshhg', 'gshhg', 'Extract data tables from binary GSHHG or WDBII data files', '', '1gmt'),
+('supplements/img/img2grd', 'img2grd', 'Extract subset of img file in Mercator or Geographic format', '', '1gmt'),
+('supplements/meca/pscoupe', 'pscoupe', 'Plot cross-sections of focal mechanisms', '', '1gmt'),
+('supplements/meca/psmeca', 'psmeca', 'Plot focal mechanisms on maps', '', '1gmt'),
+('supplements/meca/pspolar', 'pspolar', 'Plot polarities on the inferior focal half-sphere on maps', '', '1gmt'),
+('supplements/meca/psvelo', 'psvelo', 'Plot velocity vectors, crosses, and wedges on maps', '', '1gmt'),
+('supplements/mgd77/mgd77convert', 'mgd77convert', 'Convert MGD77 data to other file formats', '', '1gmt'),
+('supplements/mgd77/mgd77info', 'mgd77info', 'Extract information about MGD77 files', '', '1gmt'),
+('supplements/mgd77/mgd77list', 'mgd77list', 'Extract data from MGD77 files', '', '1gmt'),
+('supplements/mgd77/mgd77magref', 'mgd77magref', 'Evaluate the IGRF or CM4 magnetic field models', '', '1gmt'),
+('supplements/mgd77/mgd77manage', 'mgd77manage', 'Manage the content of MGD77+ files', '', '1gmt'),
+('supplements/mgd77/mgd77path', 'mgd77path', 'Return paths to MGD77 cruises and directories', '', '1gmt'),
+('supplements/mgd77/mgd77sniffer', 'mgd77sniffer', 'Along-track quality control of MGD77 cruises', '', '1gmt'),
+('supplements/mgd77/mgd77track', 'mgd77track', 'Plot track-line map of MGD77 cruises', '', '1gmt'),
+('supplements/misc/dimfilter', 'dimfilter', 'Directional filtering of 2-D gridded files in the space (or time) domain', '', '1gmt'),
+('supplements/potential/gmtgravmag3d', 'gmtgravmag3d', 'Compute the gravity/magnetic effect of a body by the method of Okabe', '', '1gmt'),
+('supplements/potential/gravfft', 'gravfft', 'Compute gravitational attraction of 3-D surfaces in the wavenumber (or frequency) domain', '', '1gmt'),
+('supplements/potential/grdgravmag3d', 'grdgravmag3d', 'Compute the gravity effect of a grid by the method of Okabe', '', '1gmt'),
+('supplements/potential/grdredpol', 'grdredpol', 'Compute the Continuous Reduction To the Pole, AKA differential RTP.', '', '1gmt'),
+('supplements/potential/grdseamount', 'grdseamount', 'Compute synthetic seamount (Gaussian or cone, circular or elliptical) bathymetry', '', '1gmt'),
+('supplements/segy/pssegy', 'pssegy', 'Plot a SEGY file on a map', '', '1gmt'),
+('supplements/segy/pssegyz', 'pssegyz', 'Create imagemasked postscript from SEGY file', '', '1gmt'),
+('supplements/segy/segy2grd', 'segy2grd', 'Converting SEGY data to a GMT grid', '', '1gmt'),
+('supplements/spotter/backtracker', 'backtracker', 'Generate forward and backward flowlines and hotspot tracks', '', '1gmt'),
+('supplements/spotter/grdpmodeler', 'grdpmodeler', 'Evaluate a plate model on a geographic grid', '', '1gmt'),
+('supplements/spotter/grdrotater', 'grdrotater', 'Finite rotation reconstruction of geographic grid', '', '1gmt'),
+('supplements/spotter/grdspotter', 'grdspotter', 'Create CVA image from a gravity or topography grid', '', '1gmt'),
+('supplements/spotter/hotspotter', 'hotspotter', 'Create CVA image from seamount locations', '', '1gmt'),
+('supplements/spotter/originator', 'originator', 'Associate seamounts with nearest hotspot point sources', '', '1gmt'),
+('supplements/spotter/rotconverter', 'rotconverter', 'Manipulate total reconstruction and stage rotations', '', '1gmt'),
+('supplements/x2sys/x2sys_binlist', 'x2sys_binlist', 'Create bin index listing from track data files', '', '1gmt'),
+('supplements/x2sys/x2sys_cross', 'x2sys_cross', 'Calculate crossovers between track data files', '', '1gmt'),
+('supplements/x2sys/x2sys_datalist', 'x2sys_datalist', 'Extract content of track data files', '', '1gmt'),
+('supplements/x2sys/x2sys_get', 'x2sys_get', 'Get track listing from the x2sys track index databases', '', '1gmt'),
+('supplements/x2sys/x2sys_init', 'x2sys_init', 'Initialize a new x2sys track database', '', '1gmt'),
+('supplements/x2sys/x2sys_list', 'x2sys_list', 'Extract subset from crossover data base', '', '1gmt'),
+('supplements/x2sys/x2sys_merge', 'x2sys_merge', 'Merge an updated COEs table (smaller) into the main table (bigger)', '', '1gmt'),
+('supplements/x2sys/x2sys_put', 'x2sys_put', 'Update track index database from track bin file', '', '1gmt'),
+('supplements/x2sys/x2sys_report', 'x2sys_report', 'Report statistics from crossover data base', '', '1gmt'),
+('supplements/x2sys/x2sys_solve', 'x2sys_solve', 'Determine least-squares systematic correction from crossovers', '', '1gmt')
 ]
 
 
--- a/doc/rst/CMakeLists.txt
+++ b/doc/rst/CMakeLists.txt
@@ -111,7 +111,7 @@ if (SPHINX_FOUND)
 			-w "${CMAKE_CURRENT_BINARY_DIR}/man.log"
 			${CMAKE_CURRENT_BINARY_DIR}/_source
 			${CMAKE_CURRENT_BINARY_DIR}/man
-			COMMAND ${GZIP} -9 -f ${CMAKE_CURRENT_BINARY_DIR}/man/*.?
+			COMMAND ${GZIP} -9 -f ${CMAKE_CURRENT_BINARY_DIR}/man/*.?gmt
 			COMMENT "Building manpages...")
 		add_dependencies (docs_man docs_depends)
 	endif (GZIP)
@@ -178,14 +178,14 @@ endif (EXISTS ${GMT_INSTALL_EXTERNAL_MAN
 install (DIRECTORY ${_man_source}/
 	DESTINATION ${GMT_MANDIR}/man1
 	COMPONENT Runtime
-	FILES_MATCHING PATTERN "*.1.gz")
+	FILES_MATCHING PATTERN "*.1gmt.gz")
 install (DIRECTORY ${_man_source}/
 	DESTINATION ${GMT_MANDIR}/man3
 	COMPONENT Runtime
-	FILES_MATCHING PATTERN "*.3.gz")
+	FILES_MATCHING PATTERN "*.3gmt.gz")
 install (DIRECTORY ${_man_source}/
 	DESTINATION ${GMT_MANDIR}/man5
 	COMPONENT Runtime
-	FILES_MATCHING PATTERN "*.5.gz")
+	FILES_MATCHING PATTERN "*.5gmt.gz")
 
 # vim: textwidth=78 noexpandtab tabstop=2 softtabstop=2 shiftwidth=2
